models/kyberswap/avalanche_c/kyberswap_avalanche_c_trades.sql

```diff
@@ -47,68 +46,65 @@ kyberswap_dex AS (
         t.evt_block_time                                                               AS block_time
         ,t.sender                                                                      AS taker
         ,t.recipient                                                                   AS maker
-        ,if(startswith(t.deltaQty0, '-'), t.deltaQty1, t.deltaQty0)                    AS token_bought_amount_raw
-        ,replace(if(startswith(t.deltaQty0, '-'), t.deltaQty0, t.deltaQty1), '-', '')  AS token_sold_amount_raw
-        ,cast(NULL as double)                                                          AS amount_usd
-        ,if(startswith(t.deltaQty0, '-'), p.token1, p.token0)                          AS token_bought_address
-        ,if(startswith(t.deltaQty0, '-'), p.token0, p.token1)                          AS token_sold_address
+        ,cast(if(starts_with(cast(t.deltaQty0 as varchar), '-'), t.deltaQty1, t.deltaQty0) as uint256)                 AS token_bought_amount_raw
+        ,cast(if(starts_with(cast(t.deltaQty0 as varchar), '-'), abs(t.deltaQty0), t.deltaQty1) as uint256)  AS token_sold_amount_raw
```

Comment:
this appears to be the table with `int256` and negative values, since it's a delta of swap output i guess? so here, we'd need to `abs()` the `deltaQty0` and `deltaQty1` every time it's outputted?
![image](https://github.com/duneanalytics/spellbook/assets/102681548/3b070904-b156-4cfa-b027-693c3f52cc96)

Reply:
looks like the same on all blockchain version scripts